app/utilities/schema.py

```diff
@@ -114,11 +117,15 @@ def load_schema_from_metadata(
         # This should not be used in production.
 
         start = time.time()
-        schema = load_schema_from_url(schema_url, language_code)
+        schema = load_schema_from_url(
+            schema_url=schema_url,
+            language_code=language_code,
+            parameters=ImmutableDict({"language": language_code}),
+        )
         duration_in_milliseconds = (time.time() - start) * 1_000
 
         cache_info = (
-            load_schema_from_url.cache_info()  # pylint: disable=no-value-for-parameter
+            load_schema_from_url.cache_info()  # pylint: disable=missing-kwoa,too-many-function-args
         )
         logger.info(
             f"load_schema_from_url took {duration_in_milliseconds:.6f} milliseconds",
```

Comment:
For the initial go live, it would be worth doing this for instrument id one also. Specifically the time, not cache info.

Reply:
Added same thing minus the cache info 👍